FWCore/Framework/src/EventProcessor.cc

```diff
@@ -182,8 +180,7 @@ namespace edm {
       //even if we have an exception, send the signal
       std::shared_ptr<int> sentry(nullptr, [areg, &md](void*) { areg->postSourceConstructionSignal_(md); });
       convertException::wrap([&]() {
-        input = std::unique_ptr<InputSource>(
-            InputSourceFactory::get()->makeInputSource(*main_input, *preg, isdesc).release());
+        input = std::unique_ptr<InputSource>(InputSourceFactory::get()->makeInputSource(*main_input, isdesc).release());
```

Comment:
```suggestion
        input = InputSourceFactory::get()->makeInputSource(*main_input, isdesc);
```
`makeInputSource()` already returns a `unique_ptr`.

Reply:
done